### ISSUE TITLE
Add degree centrality function

### ISSUE THREAD

**Issue by jhollway** [jhollway/migraph#25]
**[Original report](https://bitbucket.org/jhollway/roctopus/issue/25) by James Hollway (Bitbucket: [jhollway](https://bitbucket.org/jhollway), GitHub: [jhollway](https://github.com/jhollway)).**

----------------------------------------

**Comment by henriquesposito** [comment]
I am a bit stuck on the degree, betweenness and closeness centrality functions. The main issue I find is that I get the same error "Not a graph object" whenever I try to run any of the functions, though I cannot trace where possible issues may be with the function. For example:

```
roctopus::twomode_centrality_closeness(mpn_ryanair)
#Error in igraph::closeness(m) : Not a graph object 
``` 

I have tried to research and get around this error but I have not able to. Do you have any suggestions?

**Comment by jhollway** [comment]
Closed in #71
